examples/plot_dcrt_example.py

```diff
@@ -52,20 +51,22 @@
     y = np.maximum(0.0, y)
 
     ## dcrt Lasso ##
-    results_lasso = dcrt_zero(X, y, screening=False, verbose=True)
-    typeI_error["Lasso"].append(
-        sum(results_lasso[1][n_signal:] < alpha) / (p - n_signal)
+    selection_features, X_res, sigma2_X, y_res = dcrt_zero(X, y, screening=False)
+    variables_important_lasso, pvals_lasso, ts_lasso = dcrt_pvalue(
```

Comment:
Can we get rid of the artificial division into 2 functions ?

Reply:
I prefer, for the moment, to have a homogeneous API before changing it.

Reply:
Fair enough, but remember that for this function, the previous approach was better than the current one.